### DISCUSSION TITLE
custom system chaincode plugin not found

### DISCUSSION THREAD

**Discussion by clh123-hub** [hyperledger/fabric#2400] [Q&A]
Hello all! 
I made a system chaincode plugin databackscc.so. When starting the network, the plugin can be loaded successfully, but when invoking the plugin, it is told chaincode databackscc not found.
The experiment is based on fabric1.4.3.
Attached is the output log of the operation process.
Your suggestions are very welcome.

[The output log of the operation process.pdf](https://github.com/hyperledger/fabric/files/5982286/The.output.log.of.the.operation.process.pdf)

**Comment by guoger** [comment]
i'm not sure if `databackscc` has been properly installed and instantiated like other system chaincode. Take a look (from logs) at another system chaincode, i.e. qscc, and see if your scc follows similar pattern.

also IMHO, pastebin is probably slightly better than pdf for log sharing
